packages/app/src/components/LoadFlameModal/LoadFlameModal.tsx

```diff
@@ -64,8 +65,12 @@ function LoadFlameModal(props: LoadFlameModalProps) {
     const [fileHandle] = fileHandles
     const file = await fileHandle.getFile()
     const arrBuf = new Uint8Array(await file.arrayBuffer())
-    const newFlameDescriptor = await extractFlameFromPng(arrBuf)
-    props.respond(newFlameDescriptor)
+    const flameJsonData = await extractFlameFromPng(arrBuf)
+    try {
+      props.respond(validateExample(flameJsonData))
```

Comment:
validation should probably be in `extractFlameFromPng`

Reply:
We could add it there, but if we have different handling of failure, I would decouple this, alternative is to provide either valid object back, or undefined and list of issues, so the caller can inspect, check or provide a nice way to notify user, let me know what you think

Reply:
This was refactored a bit, take a look, but we can see perhaps later what is the best effect, I would also like to join load flame and dnd to use the same code, but currently there were some context issue if you reuse one in another.